tools/name.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+# Make sure you have python3 installed.
+# Ensure that the json_formatter is kept in Tools with this script.
+# They must be in the same folder!
+# For Windows:
+# Using command prompt type "python barrellength_volume.py"
```

Comment:
```suggestion
# Using command prompt type "python name.py"
```

Reply:
Maybe something less vague?

Reply:
if you want to change it you can, the name means nothing it's the code. easy enough to do on here.